lib/puppet/type.rb

```diff
@@ -1295,8 +1295,6 @@ def pathbuilder
   end
 
   newmetaparam(:audit) do
-    desc "(This metaparameter is deprecated and will be ignored in a future release.)
```

Comment:
It looks like you accidentally deleted the `desc` method along with the deprecation notice; that still needs to be called, wrapping the rest of the description here.